src/components/NcInputField/NcInputField.vue

```diff
@@ -161,6 +162,8 @@ const attrs = useAttrs()
 
 const inputElement = useTemplateRef('input')
 
+const isDarkTheme = useIsDarkTheme()
```

Comment:
There is no way to style the border without setting class from JS?

Reply:
How would we that work? The problem is we need to care about the browser preferred theme but also user override e.g. with theme provider. So I do not see a good solution without using the composable.

Reply:
`@media` + `[data-theme-*]` on pure styles?

It can be not the entire style property, but only a custom property with the value. Or even only the variable part. Currently, If I didn't miss anything, only `1px` changes to `-1px` in a single place.

Reply:
done